docs/modules/ROOT/pages/tutorials/logformat.adoc

```diff
@@ -0,0 +1,36 @@
+== NGINX upstream log format as JSON
```

Comment:
This is not a tutorial. For me this is a how to but a discussion would also be fine.